include/zephyr/dt-bindings/gpio/adi-sdp-120.h

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2010-2024 Analog Devices Inc.
+ * Copyright (c) 2024 Baylibre, SAS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * 
+ * @file
+ * @brief SDP-120 GPIO index definitions
+ *
+ * Defines meant to be used in conjunction with the "adi,sdp-120"
+ * ADI SDP-120 mapping.
+ *
+ * Example usage:
+ *
+ * @code{.dts}
+ * &spi1 {
+ *         cs-gpios = <&sdp_120 SDP_120_SPI_SS_N GPIO_ACTIVE_LOW>;
+ *
+ *         example_device: example-dev@0 {
+ *                 compatible = "vnd,spi-device";
+ *                 reg = <0>;
+ *         };
+ * };
+ * @endcode
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_GPIO_ADI_SDP_120_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_GPIO_ADI_SDP_120_H_
+
+/* GPIO */
+
+/**
+ * @brief IO[n] signal on a SDP-120 GPIO nexus node following
+ */
+
+#define SDP_120_IO(n) (n)
+
+	/* SPI */
+#define SDP_120_SPI_D2				SDP_120_IO(33)	/* SPI_D2 */
+#define SDP_120_SPI_D3				SDP_120_IO(34)	/* SPI_D3 */
+#define SDP_120_SERIAL_INT			SDP_120_IO(35)	/* SERIAL_INT */
+#define SDP_120_SPI_SEL_B_N			SDP_120_IO(37)	/* SPI_SEL_B_N */
+#define SDP_120_SPI_SEL_C_N			SDP_120_IO(38)	/* SPI_SEL_C_N */
+#define SDP_120_SPI_SS_N 			SDP_120_IO(39)	/* SPI_SS_N */
+
+	/* GPIO */
+#define SDP_120_GPIO0				SDP_120_IO(43) /* GPIO0 */
+#define SDP_120_GPIO2				SDP_120_IO(44) /* GPIO2 */
+#define SDP_120_GPIO4				SDP_120_IO(45) /* GPIO4 */
+#define SDP_120_GPIO6				SDP_120_IO(47) /* GPIO6 */
+
+	/* TMR */
+#define SDP_120_TMR_A				SDP_120_IO(48) /* TMR_A */
+
+	/* USART */
+#define SDP_120_UART_RX			SDP_120_IO(59) /* UART2_RX */
```

Comment:
Those are the names exposed on the connector it self. Otherwise is harder to track and refer. I use [digilent-pmod](https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/dt-bindings/gpio/digilent-pmod.h) like example and look more clear to me  that way.